pkg/handlers/v1/transformer.go

```diff
@@ -157,7 +157,8 @@ func extractTagChanges(ev configurationItemDiff) ([]TagChange, error) {
 	for k, v := range ev.ChangedProperties {
 		if !strings.HasPrefix(k, "Configuration.TagSet.") &&
 			!strings.HasPrefix(k, "SupplementaryConfiguration.TagSet.") &&
-			!strings.HasPrefix(k, "TagSet.") {
+			!strings.HasPrefix(k, "TagSet.") &&
+			!strings.HasPrefix(k, "Configuration.Tags.") {
```

Comment:
Going to cross my fingers we never find a fifth prefix

Reply:
🤞